packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/index.ts

```diff
@@ -3,3 +3,5 @@ export { default as canRegenerateImage } from './api/canRegenerateImage';
 export { default as resizeByPercentage } from './api/resizeByPercentage';
 export { default as isResizedTo } from './api/isResizedTo';
 export { default as resetImage } from './api/resetImage';
+export { OnShowResizeHandle } from './imageEditors/Resizer';
+export { X, Y } from './types/DragAndDropContext';
```

Comment:
If we export them, we may need a better name then just "X, Y"

Reply:
How about `DnDDirectionX` and `DnDDirectionY`?